tests/unit/src/main/scala/tests/BaseTypeDefinitionLspSuite.scala

```diff
@@ -0,0 +1,111 @@
+package tests
+
+import scala.meta.internal.metals.TextEdits
+import org.eclipse.{lsp4j => l}
+import scala.concurrent.Future
+import scala.meta.inputs.Input
+import scala.meta.inputs.Position
+import scala.meta.internal.mtags.MtagsEnrichments._
+
+class BaseTypeDefinitionLspSuite extends BaseLspSuite("typeDefinition") {
```

Comment:
Might not make sense to have a separate base class if this is used in one suite. Better to add those methods as private to the bottom of the suite. I don't imagine this hierachy will be used.

Reply:
Changed.